src/xform_to_datalog/datalog_facts_test.cc

```diff
@@ -72,6 +83,12 @@ claimHasTag("particle", "recipe.particle.out", "tag").
 
 // Edges:
 
+.decl grounded_dummy(x0: symbol)
```

Comment:
Why are these authorization logic facts in a section commented as `Edges:`? Is this an unintentional result of putting these facts at the end without their own comment header?

Reply:
Yes, there was no comment header for `Authorization Logic`. It should be fixed now.

Reply:
Yes, there was no header. It should be fixed now.